data/nlu.md

```diff
@@ -10,7 +10,14 @@
 - xkcd
 - show me a xkcd comic
 
+
 ## intent:hello
 - hey
 - hello
 - hi
+
+## intent:quote
+- quote
+- quote of the day
```

Comment:
I'm editing this query as it would require special handling compared to the other two. 😅